src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public String getRandomColor() {
+        int i = new Random().nextInt(Color.values().length - 1);
```

Comment:
Random must be a class field
use constants instead of numbers
don't use short name